src/KingSystem/ActorSystem/actBaseProcUnit.h

```diff
@@ -10,15 +12,18 @@ class BaseProcHandle;
 
 class BaseProcUnit {
 public:
-    bool deleteProc(void*, BaseProcHandle* handle);
+    bool deleteProc(u32, BaseProcHandle* handle);
     bool setProc(BaseProc* proc);
     void unlinkProc(BaseProc* proc);
     void cleanUp(BaseProc* proc, bool set_flag_5);
     bool isParentHandleDefault() const;
 
+    BaseProc* getProc() { return mProc; }
+    bool isReady() { return mFlags == 2; }
```

Comment:
```suggestion
    bool isReady() const { return mFlags == 2; }
```

This can be a const member function because it doesn't modify mFlags or any other member variables